src/utils/fetchQuickstarts.ts

```diff
@@ -18,22 +18,30 @@ export interface ExtendedQuickstart extends QuickStart {
   metadata: TaggedMetadata;
 }
 
+export type FetchQuickstartsOptions = {
+  'product-families'?: string[];
+  content?: string[];
+  'use-case'?: string[];
```

Comment:
@JetyAdam it looks as if we are passing these additional options through to the quickstarts API via Axios as query parameters.

It seems we are no longer using/passing along `bundle` as a parameter. However when I look at the openapi spec for quickstarts API I do not see these new params: https://github.com/RedHatInsights/quickstarts/blob/main/spec/openapi.yaml. For now this will work as the API does in fact have support for the query params in the go models - but if we ever decide to switch to the javascript-clients quickstart package - we will need to ensure the openapi spec is updated.

Reply:
Not blocking - but one other thing for us to consider here - we could make this type object a bit more JSON object friendly by changing to camel case - and then map those values in the actual call to axios back to the kebab case. IE:

```ts
export type FetchQuickstartsOptions = {
  productFamilies?: string[];
  content?: string[];
  useCase?: string[];
}
```

and in the call via axios:

```ts
 const { productFamilies, useCase, ...rest } = options || {};
  const contentPromise = axios
    .get<{ data: { content: ExtendedQuickstart }[] }>(quickstartsPath, {
      params: {
        account,
        ...(productFamilies ? { 'product-families': productFamilies } : {}),
        ...(useCase ? {'use-case': useCase} : {}),
        ...rest,
      },
    })
```

**However this adds another level of complexity as we'd need to ensure the UI type and what the service sends/receives stay in sync at all times. And you can't do the clean spread like you already have.**

It might be worth investigating migrating the service to use camel case, but that's a topic for another time.

Reply:
Great suggestion for improvement. I like that.